src/compiler/scala/reflect/quasiquotes/Reifiers.scala

```diff
@@ -130,7 +130,7 @@ trait Reifiers { self: Quasiquotes =>
       case Placeholder(Hole(tree, NoDot)) if isReifyingPatterns => tree
       case Placeholder(hole @ Hole(_, rank @ Dot())) => c.abort(hole.pos, s"Can't $action with $rank here")
       case TuplePlaceholder(args) => reifyTuple(args)
-      // Due to greediness of syntactic applied we need to pre-emptively peek inside.
+      // Due to greediness of syntactic applied we need to preemptively peek inside.
```

Comment:
I'd also looked it up and almost commented "that's a good one." Because of praeemptus, the best spelling is `preëmptively`. merriam-webster had a thread not long ago on words where the diaeresis is not optional. I said `Boötes`.